do.sh

```diff
@@ -228,6 +229,22 @@ function install_ovn_fake_multinode() {
     popd
 }
 
+function install_ovn_tester() {
+    rm -rf tester_files
+    mkdir tester_files
+    ssh_key=$(${ovn_fmn_get} ${phys_deployment} tester-node ssh_key)
+    # We need to copy the files into a known directory within the Docker
+    # context directory. Otherwise, Docker can't find the files we reference.
+    cp ${ssh_key} tester_files
+    cp ${phys_deployment} tester_files
+    ssh_key_file=tester_files/$(basename ${ssh_key})
+    phys_deployment_file=tester_files/$(basename ${phys_deployment})
+    docker build -t ovn/ovn-tester --build-arg SSH_KEY=${ssh_key_file} --build-arg PHYS_DEPLOYMENT=${phys_deployment_file} -f ${topdir}/Dockerfile .
```

Comment:
I know we mention in the readme that ovn-heater should be used with care.  But by default we copy the current user's private SSH key to a container image that will likely run on a different machine.  It makes me feel a bit uneasy.

Reply:
OK, it sounds like the best course of action for now would be not to have a default SSH key location. Users of ovn-heater will need to generate one (or point to an existing one) manually.

BTW, if we take up Ilya's suggestion of pre-creating the containers and then SSHing directly to those containers during the test, then all of this will be moot.